src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/geoMap.js

```diff
@@ -142,13 +142,23 @@ class GeoMapVisualization extends Visualization {
 
     onDataReceived(data) {
         if (!this.isInit) {
+            // FIXME: This should be simplified when issue [#1167]
+            //  (https://github.com/enso-org/ide/issues/1167) has been implemented.
+            //  Use the previous version from the history again.
```

Comment:
Also, Id keep old version in comment, as we plan to revert it soon (the mentioned issues is high-priority)

Reply:
Done.